src/GovUk.Education.ExploreEducationStatistics.Content.Model/Database/ApplicationDbContext.cs

```diff
@@ -1361,7 +1361,7 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
                     Summary =
                         "Read national statistical summaries, view charts and tables and download data files.\n\n" +
                         "Find out how and why these statistics are collected and published - [Pupil absence statistics: methodology](../methodology/pupil-absence-in-schools-in-england).",
-                    TimePeriodCoverage = TimeIdentifier.AcademicYear,
+                    TimePeriodCoverage = TimeIdentifier.SixHalfTerms,
```

Comment:
TimePeriodCoverage is a content field added by @richardpopplehive. I think it should be AcademicYear not SixHalfTerms for this Release.